testutils/clustermanager/doc.go

```diff
@@ -22,9 +22,12 @@ usage example:
 func acquireCluster() {
     clusterOps := GKEClient{}.Setup(2, "n1-standard-8", "us-east1", "a", "myproject")
     // Cast to GKEOperation
-    GKEOps := clusterOps.(GKECluster)
-    if err = GKEOps.Acquire(); err != nil {
-        log.Fatalf("Failed acquire cluster: '%v'", err)
+    GKEOps := clusterOps.(*GKECluster)
```

Comment:
```suggestion
    gkeOps := clusterOps.(*GKECluster)
```

local vars are lowercase :)

Reply:
Also for examples I strongly recommend: https://blog.golang.org/examples